client/jetpack-onboarding/steps/site-title.jsx

```diff
@@ -79,9 +76,24 @@ class JetpackOnboardingSiteTitleStep extends React.PureComponent {
 					title="Site Title ‹ Jetpack Start"
 				/>
 
-				<FormattedHeader headerText={ headerText } subHeaderText={ subHeaderText } />
+				<FormattedHeader headerText={ headerText } />
 
 				<Card className="steps__form">
+					<img
+						className="steps__illustration"
+						src={ '/calypso/images/illustrations/illustration-start.svg' }
+						alt=""
+					/>
+
+					<div className="steps__description">
+						<p>{ translate( "Let's help you get set up." ) }</p>
+						<p>
+							{ translate(
+								'First, what would you like to name your site and have as its public description?'
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'First up, what would you like to name your site and have as its public description?' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
Triple thumbs up for this recommendation @a8ci18n - fixed in https://github.com/Automattic/wp-calypso/pull/22579/commits/6c052dce810f506f6af46e7c95890624e5507e54

Reply:
Also updated the preview screenshot.